### ISSUE TITLE
官方文档示例代码统一一哈

### ISSUE THREAD

**Issue by cczxsong** [PaddlePaddle/PaddleX#15]
[https://paddlex.readthedocs.io/zh_CN/latest/quick_start.html](url)
粘贴自官方文档
`eval_dataset = pdx.datasets.ImageNet(
    data_dir='vegetables_cls',
    file_list='vegetables_cls/train_list.txt',
    label_list='vegetables_cls/labels.txt',
    transforms=train_transforms)`
[https://github.com/PaddlePaddle/PaddleX/blob/develop/tutorials/train/classification/mobilenetv2.py#L25](url)
`eval_dataset = pdx.datasets.ImageNet(
    data_dir='vegetables_cls',
    file_list='vegetables_cls/val_list.txt',
    label_list='vegetables_cls/labels.txt',
    transforms=eval_transforms)`

**Comment by jiangjiajun** [comment]
> [https://paddlex.readthedocs.io/zh_CN/latest/quick_start.html](url)
> 粘贴自官方文档
> `eval_dataset = pdx.datasets.ImageNet( data_dir='vegetables_cls', file_list='vegetables_cls/train_list.txt', label_list='vegetables_cls/labels.txt', transforms=train_transforms)`
> [https://github.com/PaddlePaddle/PaddleX/blob/develop/tutorials/train/classification/mobilenetv2.py#L25](url)
> `eval_dataset = pdx.datasets.ImageNet( data_dir='vegetables_cls', file_list='vegetables_cls/val_list.txt', label_list='vegetables_cls/labels.txt', transforms=eval_transforms)`

**Comment by cczxsong** [comment]
> > [https://paddlex.readthedocs.io/zh_CN/latest/quick_start.html](url)
> > 粘贴自官方文档
> > `eval_dataset = pdx.datasets.ImageNet( data_dir='vegetables_cls', file_list='vegetables_cls/train_list.txt', label_list='vegetables_cls/labels.txt', transforms=train_transforms)`
> > [https://github.com/PaddlePaddle/PaddleX/blob/develop/tutorials/train/classification/mobilenetv2.py#L25](url)
> > `eval_dataset = pdx.datasets.ImageNet( data_dir='vegetables_cls', file_list='vegetables_cls/val_list.txt', label_list='vegetables_cls/labels.txt', transforms=eval_transforms)`
> 
> 你好，这里是文档的问题，已经修复，谢谢反馈：）

你好，我这边看到的文档里面还没改呢

**Comment by jiangjiajun** [comment]
> > > [https://paddlex.readthedocs.io/zh_CN/latest/quick_start.html](url)
> > > 粘贴自官方文档
> > > `eval_dataset = pdx.datasets.ImageNet( data_dir='vegetables_cls', file_list='vegetables_cls/train_list.txt', label_list='vegetables_cls/labels.txt', transforms=train_transforms)`
> > > [https://github.com/PaddlePaddle/PaddleX/blob/develop/tutorials/train/classification/mobilenetv2.py#L25](url)
> > > `eval_dataset = pdx.datasets.ImageNet( data_dir='vegetables_cls', file_list='vegetables_cls/val_list.txt', label_list='vegetables_cls/labels.txt', transforms=eval_transforms)`
> > 
> > 
> > 你好，这里是文档的问题，已经修复，谢谢反馈：）
> 
> 你好，我这边看到的文档里面还没改呢

是指文档中的这部分吗https://paddlex.readthedocs.io/zh_CN/latest/quick_start.html 已经改成如下代码形式，你可以刷新下页面看看
```
train_dataset = pdx.datasets.ImageNet(
    data_dir='vegetables_cls',
    file_list='vegetables_cls/train_list.txt',
    label_list='vegetables_cls/labels.txt',
    transforms=train_transforms,
    shuffle=True)
eval_dataset = pdx.datasets.ImageNet(
    data_dir='vegetables_cls',
    file_list='vegetables_cls/train_list.txt',
    label_list='vegetables_cls/labels.txt',
    transforms=eval_transforms)
```

**Comment by cczxsong** [comment]
> > > > [https://paddlex.readthedocs.io/zh_CN/latest/quick_start.html](url) 
> > > > 粘贴自官方文档
> > > > `eval_dataset = pdx.datasets.ImageNet( data_dir='vegetables_cls', file_list='vegetables_cls/train_list.txt', label_list='vegetables_cls/labels.txt', transforms=train_transforms)`
> > > > [https://github.com/PaddlePaddle/PaddleX/blob/develop/tutorials/train/classification/mobilenetv2.py#L25](url)
> > > > `eval_dataset = pdx.datasets.ImageNet( data_dir='vegetables_cls', file_list='vegetables_cls/val_list.txt', label_list='vegetables_cls/labels.txt', transforms=eval_transforms)`
> > > 
> > > 
> > > 你好，这里是文档的问题，已经修复，谢谢反馈：）
> > 
> > 
> > 你好，我这边看到的文档里面还没改呢
> 
> 是指文档中的这部分吗https://paddlex.readthedocs.io/zh_CN/latest/quick_start.html已经改成如下代码形式，你可以刷新下页面看看
> 
> ```
> train_dataset = pdx.datasets.ImageNet(
>     data_dir='vegetables_cls',
>     file_list='vegetables_cls/train_list.txt',
>     label_list='vegetables_cls/labels.txt',
>     transforms=train_transforms,
>     shuffle=True)
> eval_dataset = pdx.datasets.ImageNet(
>     data_dir='vegetables_cls',
>     file_list='vegetables_cls/train_list.txt',
>     label_list='vegetables_cls/labels.txt',
>     transforms=eval_transforms)
> ```

不应该是将文档中的 eval_dataset 里面的file_list='vegetables_cls/train_list.txt',改成file_list='vegetables_cls/val_list.txt',吗？

**Comment by jiangjiajun** [comment]
> > > [https://paddlex.readthedocs.io/zh_CN/latest/quick_start.html](url)
> > > 粘贴自官方文档
> > > `eval_dataset = pdx.datasets.ImageNet( data_dir='vegetables_cls', file_list='vegetables_cls/train_list.txt', label_list='vegetables_cls/labels.txt', transforms=train_transforms)`
> > > [https://github.com/PaddlePaddle/PaddleX/blob/develop/tutorials/train/classification/mobilenetv2.py#L25](url)
> > > `eval_dataset = pdx.datasets.ImageNet( data_dir='vegetables_cls', file_list='vegetables_cls/val_list.txt', label_list='vegetables_cls/labels.txt', transforms=eval_transforms)`
> > 
> > 
> > 你好，这里是文档的问题，已经修复，谢谢反馈：）
> 
> 你好，我这边看到的文档里面还没改呢

原来如此，我先前没注意到这个问题，现在已经修改好了，麻烦再check一下 ：）
